seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -310,7 +409,8 @@
         "<blockquote>.blue-text {<br>&nbsp;&nbsp;color: blue;<br>}</blockquote>",
         "But remember you don't need to use a period when attaching classes to an HTML element:",
         "<code>&#60;h2 class=\"blue-text\"&#62;CatPhotoApp&#60;/h2&#62;</code>",
-        "Apply the <code>red-text</code> class to your <code>h2</code> and <code>p</code> elements."
+        "<hr>",
+        "Apply the <code>red-text</code> class to your <code>h2</code> and first <code>p</code> element."
```

Comment:
Since this is an extension of the challenge previous, I think the `red-text` class added to the h2 in the last challenge should remain in the code editor and we should make this challenge only about applying it also to the `p` tag to show it can be reused.